### PR TITLE
Remove deprecated packages

### PR DESCRIPTION
**Pull request by debo**

### CODE REVIEW

File: src/Phansible/Provider/GithubProvider.php [hunk 1]
```diff
@@ -15,15 +18,16 @@ class GithubProvider implements ServiceProviderInterface
      */
     public function register(Application $app)
```
Comment (aoga88) [inline src/Phansible/Provider/GithubProvider.php]: @debo I think we can now have return type on this

File: tests/src/Phansible/Controller/BundleControllerTest.php [hunk 2]
```diff
@@ -59,14 +59,14 @@ public function testShouldResponseWithErrorMessage()
         $data = [
             'vagrant_local' => [
                 'vm' => [
-                    'ip' => ''
-                ]
-            ]
+                    'ip' => '',
+                ],
+            ],
```
Comment (aoga88) [inline tests/src/Phansible/Controller/BundleControllerTest.php]: This as last element should not have the `,`

File: tests/src/Phansible/Controller/DefaultControllerTest.php [hunk 3]
```diff
@@ -52,13 +54,15 @@ public function testShouldRenderIndexAction()
         $this->controller->setPimple($container);
         $this->controller->indexAction();
     }
+
     /**
      * @covers \Phansible\Controller\DefaultController::docsAction
-     * @expectedException Symfony\Component\HttpKernel\Exception\NotFoundHttpException
+     * @expectException Symfony\Component\HttpKernel\Exception\NotFoundHttpException
      */
     public function testShouldThrowExceptionWhenDocFileNotExists()
```
Comment (aoga88) [inline tests/src/Phansible/Controller/DefaultControllerTest.php]:
```suggestion
    public function testShouldThrowExceptionWhenDocFileNotExists() :void
```

File: tests/src/Phansible/Roles/PhpTest.php [hunk 4]
```diff
@@ -77,7 +79,7 @@ public function testShouldGetInitialValues()
     public function testShouldNotTransformValues()
     {
         $values = [
-            'packages' => ['php5-cli', 'php5-intl']
+            'packages' => ['php5-cli', 'php5-intl'],
```
Comment (aoga88) [inline tests/src/Phansible/Roles/PhpTest.php]:
```suggestion
            'packages' => ['php5-cli', 'php5-intl']
```

File: tests/src/Phansible/Roles/PhpTest.php [hunk 5]
```diff
@@ -103,8 +105,8 @@ public function testShouldNotTransformValues()
         $expected = [
             'packages' => [
                 'php5-cli',
-                'php5-intl'
-            ]
+                'php5-intl',
```
Comment (aoga88) [inline tests/src/Phansible/Roles/PhpTest.php]:
```suggestion
                'php5-intl'
```

File: src/Phansible/Provider/GithubProvider.php (hunk 1 above)
Comment (debo) [reply src/Phansible/Provider/GithubProvider.php]: They will come don't worry, I will go through all the inspections warning after I migrated the whole app to Symfony 4

**Comment by aoga88** [comment]
@debo there a re more elements that have the same missing return type  or the last array element comma that I didn't comment, but I think I made my point about it

File: tests/src/Phansible/Controller/BundleControllerTest.php (hunk 2 above)
Comment (debo) [reply tests/src/Phansible/Controller/BundleControllerTest.php]: It's actually ok in theory... the real issue is, why are they there? I don't have them locally 😄

**Comment by debo** [comment]
I know, this is WIP PR everything will be addressed in due time

**Comment by aoga88** [comment]
@debo I haven't noticed WIP is Work In Progress 😂sorry about it

File: tests/src/Phansible/Roles/PhpTest.php (hunk 4 above)
Comment (debo) [reply tests/src/Phansible/Roles/PhpTest.php]: I genuinely don't know why this is happening, those commas aren't on my local working copy.

**Comment by debo** [comment]
No problem, it happens

**Pull request merged**